documentation/src/main/asciidoc/topics/proc_cli_modifying_cache_configuration.adoc

```diff
@@ -0,0 +1,28 @@
+[id='modifying-cache-configuration']
+= Modifying {brandname} cache configuration
+Make changes to your remote cache configuration with the {brandname} CLI.
+You can modify attributes in your cache configuration either one at a time or provide a cache configuration in XML, JSON or YAML format to modify several attributes at once.
```

Comment:
You should make this clearer if, as I understand it, the default values of attributes not included in the XML/JSON/YAML also overwrite the values set explicitly in the current configuration.

Reply:
@danberindei Do you mean if you apply custom configuration for an attribute, say "max-count", and then modify cache configuration but do not change the value of "max-count" then it results in "max-count" getting the default value?

So if users have non-default values for modifiable attributes, they must always set the non-default value when modifying configuration. If not explicitly included in the configuration, values for modifiable attributes revert to the default.

Reply:
Hey again, @danberindei. From some limited testing I'm not sure that's quite the case. With the "max-count" example I gave, if I modify an attribute like JMX statistics in an updated config but omit modifications to "max-count" I get an exception that memory configuration is required.

Reply:
@oraNod I didn't try it myself, I was just making an assumption based on the UpdateableConfigurationTest code.

Not sure what your exception is, but it sounds to me like your new updated configuration does not have a memory element and when you put that configuration, the server is trying (unsuccessfully) to reset the memory configuration to its default. 
The when-full attribute is immutable, so 2 configurations with max-count = -1 (the default) and max-count = 100 are not compatible.